pkg/logger/logger.go

```diff
@@ -171,8 +184,23 @@ func (l *Logger) PrintAuthf(username string, req *http.Request, status AuthStatu
 		Status:        string(status),
 		Message:       fmt.Sprintf(format, a...),
 	})
+	// Fallback for template errors
+	if err != nil {
+		_, ferr := fmt.Fprintf(l.writer, "%s - %s [%s] [%s] %s",
+			client,
+			username,
+			FormatTimestamp(now),
+			string(status),
+			fmt.Sprintf(format, a...))
+		if ferr != nil {
+			panic(ferr)
+		}
+	}
```

Comment:
This is meant to match the standard template right?

Reply:
Yeah, I tried to match the default templates in the 3 logging message types we have in the templating failure case.